draft-ietf-quic-recovery.md

```diff
@@ -664,8 +664,14 @@ data it can send is limited to three times the amount of data received,
 as specified in {{Section 8.1 of QUIC-TRANSPORT}}. If no additional data can be
 sent, the server's PTO timer MUST NOT be armed until datagrams have been
 received from the client, because packets sent on PTO count against the
-anti-amplification limit. Note that the server could fail to validate the
-client's address even if 0-RTT is accepted.
+anti-amplification limit.
+
+When the server receives a datagram from the client, the amplification limit is
+increased and the server resets the PTO timer.  If the PTO timer is then set to
+a time in the past, it is executed immediately. Doing so avoids sending new
+1-RTT packets prior to packets critical to the completion of the handshake.
+This could be common when 0-RTT is accepted, but the server fails to validate
```

Comment:
```suggestion
This can happen when 0-RTT is accepted but the server fails to validate
```

Reply:
It can happen without 0-RTT, it's just a lot more likely with 0-RTT.